Core/Inc/steeringio.h

```diff
@@ -34,19 +30,15 @@ typedef struct {
 	ringbuffer_t *debounce_buffer;
 	bool raw_buttons[MAX_STEERING_BUTTONS];
 	bool debounced_buttons[MAX_STEERING_BUTTONS];
+	/* Array indicating that a button has already been pressed and not unpressed */
+	bool pressed_once[MAX_STEERING_BUTTONS];
```

Comment:
This was used to make one button press = one nero index by making it so holding down a steering wheel button does not cause the button callback to be called more than once. Hence the name pressed_once since the button has already been registered as pressed once. I do not like the word pressed either but i cant think of a new name. Maybe activated? Debounced?

Reply:
debounced _button or something is better